lib/unit/models/check_payment/originate_check_payment_request.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Request to originate check payments
+# @see https://docs.unit.co/originate-check-payments/#check-payments
+module Unit
+  module CheckPayment
+    class OriginateCheckPaymentRequest
+      attr_reader :account_id, :customer_id, :customer_type, :amount, :counterparty, :description, :memo, :idempotency_key, :send_date, :tags
+
+      # @param account_id [String]
+      # @param customer_id [String]
+      # @param customer_type [String]
+      # @param amount [String]
```

Comment:
I'm wondering if amount should be an integer and  send_date should be Date.
will get back to you on this